nototools/swat_license.py

```diff
@@ -0,0 +1,319 @@
+#!/usr/bin/python
+#
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Swat copyright, bump version."""
+
+
+import argparse
+import os
+from os import path
+import re
+
+from nototools import cldr_data
+from nototools import font_data
+from nototools import noto_fonts
+
+from fontTools import ttLib
+from fontTools import misc
+
+_COPYRIGHT_ID = 0
+_VERSION_ID = 5
+_TRADEMARK_ID = 7
+_MANUFACTURER_ID = 8
+_DESIGNER_ID = 9
+_DESCRIPTION_ID = 10
+_VENDOR_URL_ID = 11
+_DESIGNER_URL_ID = 12
+_LICENSE_ID = 13
+_LICENSE_URL_ID = 14
+
+_NAME_ID_LABELS = {
+    _COPYRIGHT_ID: 'copyright',
+    _VERSION_ID: 'version',
+    _TRADEMARK_ID: 'trademark',
+    _MANUFACTURER_ID: 'manufacturer',
+    _DESIGNER_ID: 'designer',
+    _DESCRIPTION_ID: 'description',
+    _VENDOR_URL_ID: 'vendor url',
+    _DESIGNER_URL_ID: 'designer url',
+    _LICENSE_ID: 'license',
+    _LICENSE_URL_ID: 'license url',
+}
+
+_SIL_LICENSE = (
+    'This Font Software is licensed under the SIL Open Font License, '
+    'Version 1.1. This Font Software is distributed on an "AS IS" '
+    'BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express '
+    'or implied. See the SIL Open Font License for the specific language, '
+    'permissions and limitations governing your use of this Font Software.')
+
+_SIL_LICENSE_URL = "http://scripts.sil.org/OFL"
+
+_NOTO_URL = "http://www.google.com/get/noto/"
+
+
+_SCRIPT_KEYS = {
+    'Aran': 'Urdu',
+    'HST': 'Historic',
+    'LGC': ''
+}
+
+_FAMILY_KEYS = {
+  'Arimo': 'a',
+  'Cousine':'b',
+  'Tinos': 'c',
+  'Noto': 'd',
+}
+
+_CHANGES = {}
```

Comment:
I think this name should not be all caps since the dictionary is going to change.